Project3/index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="styles.css">  
+    <title>Document</title>
+    <div class="card">
+<h1>Day of Timmy</h1></head>
```

Comment:
The closing `head` tag should be after the `title` element on line 8, not here:
```suggestion
<h1>Day of Timmy</h1>
```